CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 #### Pipeline updates
 * Get MultiQC to save plots as [standalone files](https://github.com/nf-core/rnaseq/issues/183)
+* Get MultiQC to save plots as [standalone files](https://github.com/nf-core/rnaseq/issues/183): added the folder "multiqc_plots" to the output.
+* Get MultiQC to write out the software versions in a .csv file [#185] (https://github.com/nf-core/rnaseq/issues/185)
```

Comment:
Ah, I missed one thing... The little red cross means that the CI tests on Travis are failing.

You can see the first reason in the log here: https://travis-ci.org/nf-core/rnaseq/jobs/515044055#L1013-L1017 (you can click the red cross to go to Travis and find this yourself in the future).

The space between the square and round brackets breaks the markdown link. It renders as this:

> [link text] (https://link_url.com)

Instead of this:

> [link text](https://link_url.com)

```suggestion
* Get MultiQC to write out the software versions in a .csv file [#185](https://github.com/nf-core/rnaseq/issues/185)
```